### PR TITLE
Add configuration and enclave dependencies for local_attestation

### PR DESCRIPTION
**Pull request by aserdean**
Signed-off-by: Alin Gabriel Serdean <aserdean@ovn.org>

### CODE REVIEW

**Comment by andschwa** [comment]
Oh, I had WIP here, feel free to take it: https://github.com/Microsoft/openenclave/commit/358b910614af655d6411be7249878c5d07fe3098

**Comment by aserdean** [comment]
> Oh, I had WIP here, feel free to take it: [358b910](https://github.com/Microsoft/openenclave/commit/358b910614af655d6411be7249878c5d07fe3098)

Sorry @andschwa I had no idea you were working on this also!
I can close this PR if you want to tackle the issue.

**Comment by andschwa** [comment]
@aserdean Nope, you're good, I wasn't, I was just investigating and that's as far as I got. All yours!

**Comment by aserdean** [comment]
@andschwa 🙇

**Comment by andschwa** [comment]
@aserdean Is this still WIP?

File: samples/file-encryptor/CMakeLists.txt [hunk 1]
```diff
@@ -26,10 +26,10 @@ add_custom_target(sign ALL DEPENDS enclave/enclave.signed)
 
 if ((NOT DEFINED ENV{OE_SIMULATION}) OR (NOT $ENV{OE_SIMULATION}))
   add_custom_target(run
-    DEPENDS file-encryptor_host sign
+    DEPENDS file-encryptor_host sign ${CMAKE_CURRENT_SOURCE_DIR}/testfile
```
Comment (andschwa) [inline samples/file-encryptor/CMakeLists.txt]: I don't think we need `${CMAKE_CURRENT_SOURCE_DIR}` here...?

File: samples/file-encryptor/CMakeLists.txt [hunk 2]
```diff
@@ -26,10 +26,10 @@ add_custom_target(sign ALL DEPENDS enclave/enclave.signed)
...
     COMMAND file-encryptor_host ${CMAKE_CURRENT_SOURCE_DIR}/testfile ${CMAKE_BINARY_DIR}/enclave/enclave.signed)
 endif ()
 
 add_custom_target(simulate
-  DEPENDS file-encryptor_host sign
+  DEPENDS file-encryptor_host sign ${CMAKE_CURRENT_SOURCE_DIR}/testfile
```
Comment (andschwa) [inline samples/file-encryptor/CMakeLists.txt]: Ditto.

File: samples/file-encryptor/CMakeLists.txt (hunk 1 above)
Comment (aserdean) [reply samples/file-encryptor/CMakeLists.txt]: Nope. Thanks for catching it!

**Pull request merged**